### DISCUSSION TITLE
Support for ESP32 S3 Box

### DISCUSSION THREAD

**Discussion by nicoolaro** [HASwitchPlate/openHASP#471] [Hardware]
Hi ,
will there going to be support for this box: 
`https://github.com/espressif/esp-box/blob/master/docs/hardware_overview/esp32_s3_box/hardware_overview_for_box.md`

I tried M5Stack Core2 FW but says board is not supported.

**Comment by fvanroie** [comment]
The ESP32-S3 box is _not_ compatible with the M5Stack Core2. The device will need its own configuration and firmware to run properly...
I don't own the ESP32-S3 box, but I think you can make a custom configuration to make it work.

The TT21100 touch controller is a bit tricky since it is the only board to use it and needs some tweaking. However LovyanGFX supports it.
If you do get it working, please share it so other can benefit.
